docs/Automation.md

```diff
@@ -188,78 +122,22 @@ Sample `C#` code—this is interactive, but yours doesn’t need to be:
             /// </summary>
             static void Main(string[] args)
             {
-                var parameters = new Dictionary<string, string>();
-                string secondaryConfigFile = string.Empty;
-                char[] delimiters = {'='};
-                foreach (string arg in args)
-                {
-                    string[] pieces = arg.Split(delimiters);
-                    if (pieces.Length == 2)
-                    {
-                        string key = pieces[0].Trim();
-                        string value = pieces[1].Trim();
-                        if (!string.IsNullOrWhiteSpace(key) && !string.IsNullOrWhiteSpace(value))
-                        {
-                            // Special case for SecondaryConfigFile
-                            if (key.Equals("SecondaryConfigFile", StringComparison.OrdinalIgnoreCase))
-                            {
-                                secondaryConfigFile = value;
-                            }
-                            else
-                            {
-                                parameters[key] = value;
-                            }
-                            continue;
-                        }
-                    }
-                    Console.WriteLine("Ignoring malformed input: {0}", arg);
-                };
-                Console.WriteLine(StartCommand.Execute(parameters secondaryConfigFile).ToString());
-                int autoFileId = 0;
-                while (true)
-                {
-                    Console.Write("Enter process ID to capture (blank to exit): ");
-                    string input = Console.ReadLine();
-                    if (input == string.Empty)
-                        break;
-                    if (!int.TryParse(input, out int processId))
-                    {
-                        Console.WriteLine("Not a valid int: " + input);
-                        continue;
-                    }
-                    Dictionary<string, string> snapshotParameters = new Dictionary<string, string>
-                    {
-                        { CommandConstStrings.TargetProcessId, input },
-                        { CommandConstStrings.OutputFile, autoFileId++.ToString() },
-                    };
-                    Console.WriteLine(SnapshotCommand.Execute(snapshotParameters).ToString());
-                }
-                Console.WriteLine(StopCommand.Execute().ToString());
-            }
-        }
-    }
-```
+                string TestAppPath = Path.GetFullPath("../myApplication.exe");
```

Comment:
>/ [](start = 57, length = 1)

We generally use backslash (.NET framework cross-platform support is very limited)